src/styles/style.css

```diff
@@ -0,0 +1,58 @@
+* {
+  margin: 0;
```

Comment:
**[Optional]**
- [ ] It's great that you have added styling and following the wireframe :clap: It would be nice if you search some template and make your project like them as shown below :arrow_down: It will make your project look professional and will attract the recruiter :blush:

**Template**
![tempalte](https://raw.githubusercontent.com/HAMMAS-SALEEM/Leaderboard/development/src/components/images/Leaderboard-hammas.png)